src/lib/Util.ts

```diff
@@ -33,6 +33,7 @@ export class Util {
 
     public static setToIdString(aSet: Set<String>):String {
         let result = '';
+        if (aSet.size === 0 || aSet === null || aSet === undefined) return "''";
```

Comment:
@adrianlukaszek1992 był jakaś dziwny błąd w querkach, jak miałeś pusty set to się tam wstawiał pusty nawias, zrobiłem taki workaround

<img width="770" alt="Screenshot 2019-09-19 at 20 30 43" src="https://user-images.githubusercontent.com/1290312/65273339-36204900-db21-11e9-87f6-10eb30513727.png">